lib/reports/uncommitted_holdings.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require "services"
+require "reports/uncommitted_holdings_record"
+require "overlap/holding_commitment"
+
+module Reports
+  # Get holdings from spm clusters where there are items & holdings but no active commitments.
+  # Allow to optionally filter by OCN(s) and/or organization(s).
+  # "I want to know which items in HTDL lack commitments,
+  #  so I can seek new titles to secure commitments on"
+  # Invoke via bin/reports/compile_uncommitted_holdings.rb
+  class UncommittedHoldings
+    attr_reader :criteria
+    def initialize(criteria = {})
```

Comment:
Some named parameters here instead of a generic `criteria` would make this easier to follow.

Reply:
Agreed; I went in thinking this was going to be an arbitrary set of mongo where clauses or something and got confused later on, but I agree we should have a set of named parameters here instead, since there is a limited set we actually handle.